src/libraries/shims/ApiCompatBaseline.netcoreapp.netstandard.txt

```diff
@@ -429,6 +429,7 @@ MembersMustExist : Member 'protected void System.Linq.EnumerableQuery..ctor()' d
 Compat issues with assembly System.Data:
 CannotSealType : Type 'System.Data.Constraint' is effectively (has a private constructor) sealed in the implementation but not sealed in the contract.
 MembersMustExist : Member 'protected void System.Data.Constraint..ctor()' does not exist in the implementation but it does exist in the contract.
+CannotRemoveAttribute : Attribute 'System.ComponentModel.ToolboxItemAttribute' exists on 'System.Data.DataSet' in the contract but not the implementation.
```

Comment:
I don't understand this one. I see it on the DataSet impl file.

Reply:
See my comment.  I suspect its because the constructors don't match up.

Reply:
Yeah I guess this is because of the miss match, will fix: https://github.com/dotnet/standard/blob/master/src/netstandard/ref/System.Data.cs#L459

Reply:
Done